src/components/Map/LeafletMap.js

```diff
@@ -1,83 +1,97 @@
+      );
+    }
+    return (
+      <select onChange={e => handleRegionChange(e)}>
       {
-        regions.map((element, key) => <option key={key} value={key}>{element.label}</option>)
+      regions.map((element, key) => <option key={key} value={key}>{element.label}</option>)
       };
-    </select>
-  )
-}*/
+      </select>
+    );
+  }
+*/
+
+// selector named functions for lint rules makes it easier to re-ruse if needed.
+const selectedRegionSelector = (state) => state.selectedRegion.value;
+const selecteZoomSelector = (state) => state.mapProperties.zoom;
+const selecteCenterSelector = (state) => state.mapProperties.center;
 
 export default function LeafletMap() {
   const [map, setMap] = useState(null);
-  const dispatch = useDispatch()
-  const selectedRegion = useSelector((state) => state.selectedRegion.value)
-  const zoom = useSelector((state) => state.mapProperties.zoom)
-  const center = useSelector((state) => state.mapProperties.center)
-  const extent = regions[1].mapProperties.extent // conus - TODO: I hate this how can I fix this?
+  const dispatch = useDispatch();
+  const selectedRegion = useSelector(selectedRegionSelector);
+  const zoom = useSelector(selecteZoomSelector);
+  const center = useSelector(selecteCenterSelector);
+  const extent = regions[1].mapProperties.extent; // conus - TODO: I hate this how can I fix this?
 
   const classes = useStyles();
 
   const handleRegionSelectChange = (event) => {
     // Update map with new center and zoom
-    map.setView(regions[event.target.value].mapProperties.center, 
-                regions[event.target.value].mapProperties.zoom)
-    
+    map.setView(
+      regions[event.target.value].mapProperties.center,
+      regions[event.target.value].mapProperties.zoom
+    );
+
     // Update redux store with new region, zoom, and center
-    dispatch(changeRegionValue(event.target.value))
-    dispatch(changeZoom(regions[event.target.value].mapProperties.zoom))
-    dispatch(changeCenter(regions[event.target.value].mapProperties.center))
-  }
+    dispatch(changeRegionValue(event.target.value));
+    dispatch(changeZoom(regions[event.target.value].mapProperties.zoom));
+    dispatch(changeCenter(regions[event.target.value].mapProperties.center));
+  };
 
   const displayMap = useMemo(
     () => (
-      <MapContainer className = {classes.leafletContainer} 
-        center={center} 
-        zoom={zoom} 
-        scrollWheelZoom={true}
-        bounds={extent}
-        whenCreated={setMap}
-      >
-        <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
-          integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
-          crossOrigin=""/>
-        <link rel="stylesheet" href="https://unpkg.com/leaflet-draw@latest/dist/leaflet.draw-src.css" />
+      <MapContainer className = {classes.leafletContainer}
+          center={center}
```

Comment:
Is lines 68-77 spacing a result of the linter doing its thing? Personally I like 2 spaces better than 4 but I can abandon that if necessary.